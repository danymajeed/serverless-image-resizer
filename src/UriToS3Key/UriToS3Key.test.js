const UriToS3Key = require('./UriToS3Key')
const mockEvent = require('viewer-request-event')

describe('UriToS3Key', () => {
  it('should match snapshot', async () => {
    const response = await UriToS3Key(mockEvent)
    expect(response).toMatchSnapshot()
  })
})
