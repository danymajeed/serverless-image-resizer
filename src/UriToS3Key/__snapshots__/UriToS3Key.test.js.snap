// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UriToS3Key should match snapshot 1`] = `
{
  "clientIp": "108.41.158.64",
  "headers": {
    "accept": [
      {
        "key": "Accept",
        "value": "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9",
      },
    ],
    "accept-encoding": [
      {
        "key": "Accept-Encoding",
        "value": "gzip, deflate",
      },
    ],
    "accept-language": [
      {
        "key": "Accept-Language",
        "value": "en-US,en;q=0.9",
      },
    ],
    "cache-control": [
      {
        "key": "Cache-Control",
        "value": "max-age=0",
      },
    ],
    "host": [
      {
        "key": "Host",
        "value": "d1ppm5fhf7dvj2.cloudfront.net",
      },
    ],
    "upgrade-insecure-requests": [
      {
        "key": "Upgrade-Insecure-Requests",
        "value": "1",
      },
    ],
    "user-agent": [
      {
        "key": "User-Agent",
        "value": "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36",
      },
    ],
  },
  "method": "GET",
  "querystring": "nextExtension=webp&height=&sourceImage=/ghosted.jpg&width=300",
  "uri": "/300/ghosted.webp",
}
`;
